src/AmpWpPlugin.php

```diff
@@ -105,7 +108,11 @@ protected function get_bindings() {
 	 *                      to argument values.
 	 */
 	protected function get_arguments() {
-		return [];
+		return [
+			ServerTiming::class => [
+				'verbose' => filter_input( INPUT_GET, QueryVar::VERBOSE_SERVER_TIMING, FILTER_VALIDATE_BOOLEAN ),
```

Comment:
"is logged in" condition is missing here.

Reply:
Resolved via https://github.com/ampproject/amp-wp/pull/5005/commits/3c274a5e17494dfd6ace3728833f80b567d8124e